src/test/java/org/prebid/server/handler/SetuidHandlerTest.java

```diff
@@ -66,14 +72,17 @@
 
     @Before
     public void setUp() {
+        given(gdprService.analyze(any(), any(), any()))
```

Comment:
It looks like this test could be improved to cover different grpr failure scenarios.